src/lib/travis.js

```diff
@@ -20,7 +20,7 @@ const travisyml = {
   },
   // https://github.com/nodejs/Release#release-schedule
   node_js: ['9', '8', '6', '4'], // eslint-disable-line camelcase
-  after_success: ['npm run semantic-release'], // eslint-disable-line camelcase
+  after_success: ['npm install -g travis-deploy-once', 'travis-deploy-once "npm run semantic-release"'], // eslint-disable-line camelcase
```

Comment:
as mentioned in the other PR, I would do sth like `npm install -g travis-deploy-once@3`

Reply:
I set it to `npm install -g travis-deploy-once@3` as the travis-deploy-once CLI will exist only in version 4, after semantic-release/travis-deploy-once#38 is merged